web/src/pages/mgmt/cms.tsx

```diff
@@ -175,19 +179,15 @@ const registerPreview = (
 
 const Admin = (): ReactElement => {
   useMountEffect(() => {
```

Comment:
[dust] - So you removed the return here. Doesn't that change how this works fundamentally?

For useEffect call isn't the return the clean up function?

So is this intentional? Or Am I misreading how useMountEffect works